Sources/Components/Profile/IdentityView.swift

```diff
@@ -75,13 +86,42 @@ public class IdentityView: UIView {
         return imageView
     }()
 
+    private lazy var profileNameWrapperView: UIView = {
+        let view = UIView(withAutoLayout: true)
```

Comment:
It would be nice to call this let for `wrapperView` or something else than `view`.
Just my subjective taste on that matter.